src/astro/sql/operators/agnostic_load_file.py

```diff
@@ -141,6 +150,38 @@ def _load_dataframe(self, path):
                 stream, **deserialiser_params.get(file_type, {})
             )
 
+    def get_paths(self, path, file_conn_id):
+        url = urlparse(path)
+        file_location = url.scheme
+        return {
+            "s3": self.get_paths_from_s3,
+            "gs": self.get_paths_from_gcs,
+            "http": lambda url, file_conn_id: [urlunparse(list(url))],
+            "https": lambda url, file_conn_id: [urlunparse(list(url))],
```

Comment:
Why are we using `urlparse` on line 154 and then `urlunparse` here?

Reply:
We have to get the scheme to identify the file location to do that we are parsing url via urlparse and then all the methods are accepting the same set of params, so to keep the pattern consistent.